g15helper/g15helper.rc.in

```diff
@@ -20,8 +20,8 @@ IDI_ICON1 ICON DISCARDABLE "@G15HELPER_ICON@"
 #endif
 
 VS_VERSION_INFO VERSIONINFO
-	FILEVERSION		@PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,0
-	PRODUCTVERSION	@PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,0
+	FILEVERSION		@PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
```

Comment:
Why is it called "TWEAK" here when we refer to it as "build number" seemingly everywhere else? :thinking:

Reply:
https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION_TWEAK.html